tests/test_sshfsmount.cpp

```diff
@@ -39,10 +47,10 @@ struct SshfsMount : public mp::test::SftpServerTest
         channel_is_closed.returnValue(0);
     }
 
-    mp::SshfsMount make_sshfsmount()
+    mp::SshfsMount make_sshfsmount(mp::optional<std::string> target = mp::nullopt)
     {
         mp::SSHSession session{"a", 42};
-        return {std::move(session), default_source, default_target, default_map, default_map};
+        return {std::move(session), default_source, target.value_or(default_target), default_map, default_map};
```

Comment:
If `std::optional::value` is unsupported on macOS, I doubt this will be…

Reply:
Indeed, this works on macos.